### PR TITLE
Feature/dgi 92

### PR DESCRIPTION
**Pull request by MichaelJeppesen**
- Added a hook to get methods that format numbers in different ways
 - Added methods to format numbers with thousand separators, one which includes decimals, and one which does not
 - Added a method to convert a formatted string-number back into a number (as when using "." as thousand-separator it is seen as a decimal and converts to the wrong number with parseInt)
- formatted the InputDKK with thousand-separator without decimal
- formatted the TaxTotal with thousand-separator with decimal

### CODE REVIEW

File: frontend/src/components/Statement/TaxTotal.tsx [hunk 1]
```diff
@@ -11,10 +12,11 @@ interface Props {
 
 const TaxTotal: FC<Props> = ({ name, tax }) => {
   const { form } = useContext(FormControlContext);
+  const { formatNumberThousandSepDecimal } = useFormatNumber();
 
   const total = useMemo(() => {
     if (!form) return "";
-    return ((form[name] as number) * tax) / 1000;
+    return formatNumberThousandSepDecimal(((form[name] as number) * tax) / 1000);
```
Comment (objmj-itminds) [inline frontend/src/components/Statement/TaxTotal.tsx:19]: Lets use 

``` 
<NumberFormat
              decimalSeparator=","
              thousandSeparator="."
              prefix={"$"}
            />```

**Review by objmj-itminds — CHANGES_REQUESTED** [review]
I still feel like we can drop the custom hook completely.

**Comment by MichaelJeppesen** [comment]
> 

I cant seem to format the NumberFormat element as i want
cannot set bgColor, rounded corners, it always just ends up being a white box, with white text

Screenshot 2022-01-19 104623
i can only see the text because i have marked it

it should be possible to add a renderText property which should be how it is rendered, but when i set bgColors, roundLeft etc. on that element, nothing changes

**Comment by objmj-itminds** [comment]
I'll take over this branch.

**Pull request closed without merging**